examples/src/multiple_kms_cmk_regions.py

```diff
@@ -0,0 +1,48 @@
+# Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You
+# may not use this file except in compliance with the License. A copy of
+# the License is located at
+#
+# http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is
+# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+"""Example showing basic encryption and decryption of a value already in memory using multiple KMS CMKs in multiple regions."""
+import aws_encryption_sdk
+
+
+def encrypt_decrypt(key_arn1, key_arn2, region_name1, region_name2, source_plaintext, botocore_session=None):
```

Comment:
I would the encrypt and decrypt functionality to two different functions, so that it is more readable and can be re-used for encryption and decryption under the different keys, if needed. 

Refer:  https://github.com/aws/aws-encryption-sdk-c/blob/master/examples/kms_discovery.cpp to give you an idea on what are are looking for in this example.